aptos-move/mvhashmap/src/versioned_aggregators.rs

```diff
@@ -175,33 +177,33 @@ impl VersionedValue {
     fn read_latest_committed_value(
         &self,
         next_idx_to_commit: TxnIndex,
-    ) -> Result<u128, MVAggregatorsError> {
+    ) -> Result<AggregatorValue, MVAggregatorsError> {
         use AggregatorEntry::*;
 
         self.versioned_map
             .range(0..next_idx_to_commit)
             .next_back()
-            .map_or(
-                self.base_value.ok_or(MVAggregatorsError::NotFound),
+            .map_or_else(
+                || self.base_value.clone().ok_or(MVAggregatorsError::NotFound),
                 |(_, entry)| match &**entry {
-                    Value(v, _) => Ok(*v),
-                    Snapshot(_, _) | Delta(_) => unreachable!(
-                        "Snapshot or Delta entries may not exist for committed txn indices"
-                    ),
+                    Value(v, _) => Ok(v.clone()),
+                    Apply(_) => {
+                        unreachable!("Apply entries may not exist for committed txn indices")
+                    },
                     Estimate(_) => unreachable!("Committed entry may not be an Estimate"),
                 },
             )
     }
 
-    // Gate the estimate bypass logic by whether that functionality is enabled
-    // (read_estimate_deltas flag) for the aggregator.
-    fn applicable_bypass(&self, bypass: &EstimatedEntry) -> EstimatedEntry {
-        if self.read_estimate_deltas {
-            *bypass
-        } else {
-            EstimatedEntry::NoBypass
-        }
-    }
+    // // Gate the estimate bypass logic by whether that functionality is enabled
```

Comment:
Why comment out?

Reply:
because I needed it in the match together with types that are not estimates, and so couldn't use this function